design/pipeline-artifacts.md

```diff
@@ -95,31 +100,31 @@ You can control the location where artifacts are downloaded using the `downloadR
 - job: Build
   steps:
   - script: dotnet publish --configuration $(buildConfiguration)
-  - artifact: bin/*
+  - upload: bin/*
 - job: Deploy
   steps:
-  - downloadArtifact:
+  - download:
```

Comment:
What is the default root? When referencing a pipeline resource, is it \<PIPELINE_ALIAS\>/\<ARTIFACT_NAME\>?

What about for the current pipeline? Is it just the \<ARTIFACT_NAME\>? And we don't need to prefix with `self` or anything because the likelyhood of collision is low?

Reply:
Great question. Clarified on Line 43.

Reply:
Not sure this clarified it for me. Eric pointed out that although you can't have two artifacts with the same name in "this" pipeline, you certainly can as soon as you start pulling in other pipelines. We'd be smarter/better off to simply namespace all the artifacts (using folders named for the pipeline) from the get-go. This will be a breaking change (or at least needlessly complicated and confusing) if we introduce it later.

Reply:
Chatted IRL - we'll keep this design for now, and assume that likelihood of collision is low.